### PR TITLE
Update uk-wellcome-campus.md

### PR DESCRIPTION
**Pull request by priyanka-surana**
I pretty much copied from the London node.

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1668"></a>

### CODE REVIEW

File: markdown/events/2023/hackathon-march-2023/uk-wellcome-campus.md [hunk 1]
```diff
@@ -1,19 +1,67 @@
...
+location_latlng: [52.079047, 0.187607]
 ---
 
-Local event to be held at the Wellcome Trust Genome Campus, Cambridge.
+Local event to be held at the [Wellcome Genome Campus, Cambridge](https://goo.gl/maps/k3fuhxd6R8Tnuoay8).
 
 Primary contact: [<i class="fab fa-slack"></i> Priyanka Surana](https://nfcore.slack.com/team/U02JA08N0BC)
 
-_Details to be confirmed_
+Please use the [main registration form](https://nf-co.re/events/2023/hackathon-march-2023) to sign up.
```
Comment (christopher-hakkaart) [inline markdown/events/2023/hackathon-march-2023/uk-wellcome-campus.md:19]:
```suggestion
Site registration is now closed.
```

**Review by christopher-hakkaart — APPROVED** [review]
Site registrations are closed but otherwise looks good

**Comment by priyanka-surana** [comment]
Accepted the suggestion. Thanks 👍 
The deploy website check fails, what changes are needed? I do not speak PHP.

**Comment by mashehu** [comment]
My guess the failure is due to the new gh action runners, we are trying out at the moment

**Comment by priyanka-surana** [comment]
> My guess the failure is due to the new gh action runners, we are trying out at the moment

So should I merge it anyway?

**Comment by mashehu** [comment]
Wait maybe a bit more, just to be sure.

**Pull request merged**